CARAMAN GHENADIE

GhennaC opened the pull request.

GhennaC committed:
hello world

GhennaC committed:
code challenge1,2,Caraman Ghenadie

miualinionut commented on _1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java:
```diff
@@ -0,0 +1,29 @@
+        fizz = "Fizz";
+        rizz = "Rizz";
+        jazz = "Jazz";
+        for(int i = 1; i <= 100 ; i++){
```
pentru numarul 105, care este divizibil si cu 3,5,7 afiseaza FizzBuzzRizz ?

GhennaC replied:
nu am incercat ,dar presupun ca face asta : ))

miualinionut commented on _1_basics/src/main/java/code/_4_student_effort/FooBarQix.java:
```diff
@@ -0,0 +1,41 @@
+
+public class FooBarQix {
+
+    public void containsDigit(StringBuffer result,int number){
```
bravo

GhennaC committed:
code challenge 1,2

miualinionut commented on _1_basics/src/main/java/code/_4_student_effort/BoxingMatch.java:
```diff
@@ -0,0 +1,23 @@
+package code._4_student_effort;
+
+public class BoxingMatch {
```
bravo

miualinionut commented on _1_basics/src/main/java/code/_4_student_effort/TestStudioDataBase.java:
```diff
@@ -0,0 +1,74 @@
+        return names;
+    }
+    public String[] getStudiosWithActor(Studio[] studios){
+        String[] names = new String[10];
```
bravo

peste 2 cursuri vom invata la colectii cum sa folosim liste care se redimensioneaza singure

GhennaC committed:
2 design patterns

GhennaC committed:
2 project, rog sa verificati,
feedback: a fost foarte ciudat structurat... am stat vreo jumatate de ora sa inteleg in care folder trebuie sa modific... + un test din requirements din chessboard a fost diferit decat din solution,in req era setat pe true,in sol era pe false. nu am inteles cerintele...
